### ISSUE TITLE
Popup that says Orbot not installed

### ISSUE THREAD

**Issue by bmml1** [LN-Zap/zap-android#261]
##  Description

When trying to connect to my lnd node via tor a popup appears which states that Orbot is not installed even though it is.

## Expected Behavior

Connect to LND Node via tor with orbot in vpn mode.

## Actual Behavior

Popup with the above message. When canceling the popup there seems to be an endless "loading loop"

## Possible Fix

It worked with the release before. So there where probably some changes.

## Your Environment
- Orbot Version: Latest from F-Droid Guardian Project Official Releases Source (16.3.0-RC-1-tor-0.4.2.7(Tor 0.4.2.7-openssl1.1.1g)
- Zap version: 0.3.4-beta
- Android version: Android 11(Graphene OS)
- Device: Pixel 3a
- LND Version: 0.11.1

**Comment by michaelWuensch** [comment]
Hi,
Thanks for the report.

Did you just update Zap, or did you change anything else like updating lnd or your android version?
I didn't change anything regarding handling with tor in the last update. The only thing that might have an effect is that I increased the target sdk to 30.

Right now I am still running lnd v0.10.4
Hopefully I will find time on the weekend to update and investigate the issue.

**Comment by bmml1** [comment]
I recently updated all three.
Graphene OS release for Android 11 was maybe 2-3 weeks ago.
A few days ago I updated to lnd v0.11.1

A connection to Electrum hidden service still works(if that helps)

Let me know if I can help anyhow.

**Comment by michaelWuensch** [comment]
Hi, 
I was able to track it down. It was actually a combination of the Zap update and the Android update.
Increasing the target SDK to 30 caused the new package visiblity restrictions of Android 11 to kick in on Android 11 devices.
So the app can't see that Orbot is already installed on these devices.
I have fixed this now and will make a new release in the coming days.
Thanks again for the report!
